pyproject.toml

```
@@ -21,7 +21,7 @@ task = 'taskipy.cli:main'
 
 [tool.poetry.dependencies]
 python = "^3.6"
-toml = "^0.10.0"
+tomli = "1.2.3"
```

Let's not version lock, and instead use semver :)
```suggestion
tomli = "^1.2.3"
```

---

Agreed, that must have been an accident on my part. I’ll get that fixed 🙂